requirements.txt

```diff
@@ -12,4 +12,5 @@ pytz==2016.7
 psycopg2==2.6.2
 uWSGI==2.0.14
 wsgiref==0.1.2
+factory-boy==2.7.0
```

Comment:
Fixed in https://github.com/cellcounter/cellcounter/pull/446/commits/c943a5c271046e85985069f296d6d23fab98dd31